spharpy/spherical.py

```diff
@@ -248,6 +248,104 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
+def renormalize(data, channel_convention, current_norm, target_norm, axis=-2):
+    """
+    Renormalize either an array of SH coefficients or SH basis functions.
+
+    Parameters
+    ----------
+    data : ndarray
+        Data which should be renormalized, either SH
+        coefficients or SH bases
+    channel_convention : str
+        Channel convention of the data which should be renormalized
```

Comment:
Here and below, I would explicitly mention the options, since it is now a public function.